app/pages/index.tsx

```diff
@@ -42,19 +61,189 @@ const Home: NextPage = () => {
+    setSelectedBinType(event.target.value);
+  };
+
+  const handleBinVolumeChange = (event: SelectChangeEvent<string>) => {
+    setSelectedBinVolume(event.target.value);
+  };
+
+  const toggleDrawer = (open: boolean) => () => {
+    setDrawerOpen(open);
+  };
+
+  console.log(sensors);
+
   return (
     <DefaultLayout>
       <div className="content-wrapper">
         <Sidebar sensors={sensors} />
         <div id="map" style={{ height: "75vh", width: "100%" }}>
-          <Map sensors={sensors} fillLevelThreshold={79} />
+          <Map 
+            sensors={sensors}
+            alertThreshold={50}
+            filterThresholdMinimum ={thresholdRange[0]}
+            filterThresholdMaximum={thresholdRange[1]}
+            selectedGroup={selectedGroup}
+            selectedAssetTag={selectedAssetTag}
+            selectedBinType={selectedBinType}
+            selectedBinVolume={selectedBinVolume}
+          />
         </div>
+        <Drawer
+          anchor="left"
+          open={drawerOpen}
+          onClose={toggleDrawer(false)}
+          variant="persistent"
+          className={`drawer ${drawerOpen ? "open" : ""}`}
+        >
+          <div className="drawer-content">
+            <Typography variant="h6">Create Route</Typography>
+            <Typography id="slider-label">Fill Level Threshold</Typography>
+            <Slider
+              value={thresholdRange}
+              onChange={handleThresholdChange}
+              valueLabelDisplay="auto"
+              min={0}
+              max={100}
+              valueLabelFormat={(value) => `${value}%`}
+              marks={[
+                { value: 0, label: "0%" },
+                { value: 50, label: "50%" },
+                { value: 100, label: "100%" },
+              ]}
+              step={1}
+            />
+            <div className="form-control with-margin">
+              <FormControl fullWidth>
+                <InputLabel id="group-label">Group</InputLabel>
+                <Select
+                  labelId="group-label"
+                  id="group-select"
+                  value={selectedGroup}
+                  onChange={handleGroupChange}
+                >
+                  <MenuItem value=""> - </MenuItem>
+                  <MenuItem value="Alton North">Alton North</MenuItem>
+                  <MenuItem value="Halton West">Halton West</MenuItem>
+                  <MenuItem value="Bolton South">Bolton South</MenuItem>
+                </Select>
+              </FormControl>
+            </div>
+            <div className="form-control with-margin">
+              <FormControl fullWidth>
+                <InputLabel id="asset-tag-label">Asset Tag</InputLabel>
+                <Select
+                  labelId="asset-tag-label"
+                  id="asset-tag-select"
+                  value={selectedAssetTag}
+                  onChange={handleAssetTagChange}
+                >
+                  <MenuItem value=""> - </MenuItem>
```

Comment:
Where is this list of tag options from?

Reply:
these were just arbitrary strings I picked because I don't know what the data for asset tags will be. They're flavors of quarks